.travis.yml

```diff
@@ -90,54 +96,92 @@ matrix:
     env: TEST_CHECK_FILES=true
     compiler: clang
     name: test check files
-    # Run full builds after static analysis
-  - os: linux
+
+  # Run full builds after static analysis
+  - &GccDebug
+    os: linux
     compiler: gcc
     dist: trusty
-    env: BUILD_TYPE=Debug COVERAGE=ON
-    name: gcc debug with coverage
-  - os: linux
+    env: BUILD_TYPE=Debug COVERAGE=OFF
+    name: gcc debug
+    stage: Build Libraries
```

Comment:
can we do the Build Libraries stage for mac os x?

Reply:
Something with brew and travis broke since I put in this PR. Offline discussion between @kidder and I we came to decision to leave for now and revisit after this is in.